schemas/general/SCHEMA.md

```diff
@@ -0,0 +1,70 @@
+# General Schema
+
+Table definitions for the generic schema. These tables can be used for any protocol.
+
+## Schema
+
+### Incentive Claim Data
+
+| Property              | Description                                                     | Type   |
+|-----------------------|-----------------------------------------------------------------|--------|
+| timestamp             | The timestamp of the claim.                                     | number |
+| chain_id              | The standard chain id.                                          | number |
+| user_address          | The address of the user who claimed the incentives.             | string |
+| claimed_token_address | The smart contract address of the claimed token.                | string |
+| claimed_token_amount  | The amount of the token claimed, decimal normalized.            | number |
+| claimed_token_usd     | The amount of claimed tokens in USD.                            | number |
+| other_incentive_usd   | Any incentives outside of the claimed token, in this transaction, summed up in USD terms. | number |
+
+### Airdrop
+
+| Property                        | Description                                                     | Type   |
+|---------------------------------|-----------------------------------------------------------------|--------|
+| airdrop_timestamp               | The timestamp the airdrop was given to the user.                | number |
+| user_address                    | The address of the user claiming the airdrop.                   | string |
+| claim_timestamp                 | The timestamp of when the user claimed the airdrop.             | number |
+| airdrop_token_address           | The smart contract address of the airdropped token.             | string |
+| airdrop_token_symbol            | The symbol of the token being airdropped.                       | string |
+| token_amounts                   | The amount of each token airdropped, decimal normalized.        | [number] |
+| amount_usd                      | The USD value of the airdropped tokens.                         | number |
+| first_outgoing_transaction_address | The address of the first receiver of airdropped tokens.      | string |
+| first_outgoing_transaction_timestamp | The timestamp of the first outgoing transaction.          | number |
+| first_outgoing_transaction_amount | The amount of the first outgoing transaction, decimal normalized. | number |
```

Comment:
might be really difficult to get protocols to provide us this?

Reply:
I agree, I am removing. I think The idea of this table is a little skewed. These would be derived fields.

I am updating the table so that it is more about airdrop claims on a transaction level